charts/ecr-viewer/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: ecr-viewer
+description: A Helm chart for the DIBBs eCR Viewer
+type: application
+version: 0.1.0
+appVersion: "1.1.10"
```

Comment:
Not sure if it really matters but the app version doesn't match what's in ingestion